specs/protocol/span-batches.md

```diff
@@ -133,16 +133,15 @@ tx_sigs ++ tx_tos ++ tx_datas ++ tx_nonces ++ tx_gases ++ protected_bits`
 [EIP-1559]: https://eips.ethereum.org/EIPS/eip-1559
 [EIP-155]: https://eips.ethereum.org/EIPS/eip-155
 
-### Max span-batch size
+### Span Batch Size Limits
 
-Total size of encoded span batch is limited to `MAX_SPAN_BATCH_SIZE` (currently 10,000,000 bytes,
-equal to `MAX_RLP_BYTES_PER_CHANNEL`). Therefore every field size of span batch will be implicitly limited to
-`MAX_SPAN_BATCH_SIZE` . There can be at least single span batch per channel, and channel size is limited
-to `MAX_RLP_BYTES_PER_CHANNEL` and you may think that there is already an implicit limit. However, having an explicit
-limit for span batch is helpful for several reasons. We may save computation costs by avoiding malicious input while
-decoding. For example, let's say bad batcher wrote span batch which `block_count = max.Uint64`. We may early return
-using the explicit limit, not trying to consume data until EOF is reached. We can also safely preallocate memory for
-decoding because we know the upper limit of memory usage.
+Total size of encoded span batch is limited to `MAX_RLP_BYTES_PER_CHANNEL` which is defined in the
+[Protocol Parameters table](#protocol-parameters).
+This is done at the channel level rather than at the span batch level.
+
+In addition to the byte limit, the number of blocks, transactions per block, and total transactions is limited
```

Comment:
If we already limit the number of blocks and total transactions by this limit, don't we implicitly already limit the transactions per block with it? So we could drop mentioning it explicitly.